db/migrate/20231009132343_devise_invitable_add_to_users.rb

```diff
@@ -0,0 +1,26 @@
+class DeviseInvitableAddToUsers < ActiveRecord::Migration[7.0]
```

Comment:
My initial thought without considering the gem would be to separate Invitations from Users and have an Invitations table/model. This would be more maintainable in the long run. 

However, looking at the gem, I see that adding to Users table is a 'normal' practice. I was thinking it doesn't make sense if only Org users use invites, as then we'd have a User class that for adopters has completely irrelevant fields. But, then I thought we could also leverage invitations between adopters as well at some point. 

This is not to say this needs to change. I think it works like this and is ok a. But, I think with more time and a bigger budget I'd go for separating those concerns :D

Reply:
I totally agree with your point and would love to make those changes too, but sadly, this gem doesn't let us do that (there is even an issue for this: https://github.com/scambra/devise_invitable/issues/888). 

Let's go with what we have for now and think about extracting it later on if needed.